src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+import Icon from '../Icon/Icon';
+/**
+ * The dropdown menu allow user to customize the button text, icon and action.
+ */
+const Dropdown = ({ className, items, topIcon, children, open }) => {
+  const dropdownClass = classes('Dropdown', { 'Dropdown--open': open }, className);
+  return (
+    <div className={dropdownClass}>
+      <button type="button" className="Dropdown--button" tabIndex="0">
+        {topIcon && <Icon width={16} height={16} kind={topIcon} />}
+        <p>{children}</p>
```

Comment:
I think we either need to check if `children` is a component or text or make the a render prop. Otherwise if we add a component as a child it will get wrapped in a `p` tag which isn't semantic for non text children